plugins/com.google.cloud.tools.eclipse.dataflow.ui/helpContexts.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE contexts [
+
+  <!ENTITY group-id "&lt;b&gt;Group ID:&lt;/b&gt; The Maven Group ID. Should be an alphanumeric path separated by periods. Typically the same as the Java package name.">
+
+  <!ENTITY artifact-id "&lt;b&gt;Artifact ID:&lt;/b&gt; The Maven Artifact ID. Should be an alphanumeric name separated by dashes.">
+
+]>
+<contexts>
+  <context id="NewDataflowProjectContext" title="Create Google Cloud Dataflow Project">
+    <description>This wizard creates a new Eclipse project for Google Cloud Dataflow.
+
+      &group-id;
+
+      &artifact-id;
+      
+      &lt;b&gt;Project template:&lt;/b&gt; @tgroh please suggest text
```

Comment:
'The initial contents of the generated project.'

'_starter pipeline_ contains a simple pipeline which logs its input'

'_example pipelines_ contains the suite of Dataflow Word Count examples.' (this should probably have a link and a bit of extra text describing it to https://beam.apache.org/get-started/wordcount-example/ or https://cloud.google.com/dataflow/examples/wordcount-example

Reply:
Done, though it seems we can't include links here.